src/alfasim_score/common.py

```diff
@@ -61,6 +70,18 @@ def prepare_for_regression(values: Dict[str, Any]) -> Dict[str, Any]:
 
 
 def convert_quota_to_tvd(quota: Scalar, air_gap: Scalar) -> Scalar:
-    """Convert quota value to TVD given the air_gap"""
+    """Convert quota value to TVD given the air gap."""
     tvd = np.abs(quota.GetValue(LENGTH_UNIT)) + air_gap.GetValue(LENGTH_UNIT)
     return Scalar(tvd, LENGTH_UNIT)
+
+
+def convert_api_gravity_to_oil_density(api_gravity: Scalar) -> Scalar:
+    """Calculate the oil standard condition density based on API density."""
+    return Scalar(141.5 / (api_gravity.GetValue(FRACTION_UNIT) + 131.5), DENSITY_UNIT)
```

Comment:
Are you sure that it is the oil density?

Reply:
What's wrong here? The API gravity refers to the liquid phase, not only the oil, is it? I've seen references saying "liquid petroleum products", maybe I understood this wrongly.

Reply:
It is calculating the SG, that must be converted to density.
https://en.wikipedia.org/wiki/API_gravity

Reply:
Ok, got it. Missing the product with water density in standard condition.